Source/WebCore/ChangeLog

```diff
@@ -1,3 +1,34 @@
+2022-05-10  Yusuke Suzuki  <ysuzuki@apple.com>
+
+        Rename EventTrackingRegions::Event to EventTrackingRegions::EventType
+        https://bugs.webkit.org/show_bug.cgi?id=240295
+
+        Reviewed by NOBODY (OOPS!).
+
+        This patch is follow-up after r293967 by Darin's comment. EventTrackingRegions::Event is not event actually,
+        it is just an EventType. This patch renames it with EventType. We also rename variables "event" to "eventType".
```

Comment:
Also, we have a patch for internal. Please check rdar://93075115 too (it has a link to that).